### ISSUE TITLE
create-payload-app not installing into Next.js 14 app

### ISSUE THREAD

**Issue by Jose-henry** [payloadcms/payload#7430]
### Payload Version

3.0 beta

### Node Version

22.5.1

### Next.js Version

14

### Describe the Bug

npx create-payload-app@beta  is not being integrated into existing nextjs app

### Reproduction Steps

tried to integrate payload 3.0 into an existing project, but it seems to always ask for project name to create new project

### Adapters and Plugins

db-mongodb

**Comment by denolfe** [comment]
Payload only supports specific Next.js 15 versions. Because you are on Next 14, it will not be able to install inside of it.

That being said, I will be adding a check to notify the user that this is the case.

**Comment by Jose-henry** [comment]
I upgraded to 15.0.0-canary.53 and react to 19.0.0-rc-6230622a1a-20240610 but same thing

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by denolfe** [comment]
I've tested this by using the following commands, and it properly detects the Next app:

```
pnpm create next-app@canary --yes
cd <dir-name>
mkdir src/app/\(app\) && mv src/app/*.* src/app/\(app\)
pnpm create payload-app@beta
```

**Comment by Jose-henry** [comment]
Alright thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by denolfe** [comment]
Closed by #7434
